### ISSUE TITLE
DatePicker : should be controllable via keyboard

### ISSUE THREAD

**Issue by EugenFischer** [primefaces/primefaces#11087]
### Description

For accessibility reasons, it is necessary that the DatePicker can be operated completely via the keyboard.
At least the same possibilities that are already available in jQuery Calender should be supported by the DatePicker.

### Describe the solution you would like

The opened DatePicker should be controllable via keyboard.
